databricks/koalas/strings.py

```diff
@@ -350,3 +360,717 @@ def isdecimal(self) -> 'ks.Series':
             lambda x: x.str.isdecimal(),
             BooleanType()
         ).alias(self.name)
+
+    def cat(self, others=None, sep=None, na_rep=None, join=None) -> 'ks.Series':
+        """
+        Not supported.
+        """
+        raise NotImplementedError()
+
+    def center(self, width, fillchar=' ') -> 'ks.Series':
+        """
+        Filling left and right side of strings in the Series/Index with an
+        additional character. Equivalent to :func:`str.center`.
+
+        Parameters
+        ----------
+        width : int
+            Minimum width of resulting string; additional characters will be
+            filled with fillchar.
+        fillchar : str
+            Additional character for filling, default is whitespace.
+
+        Returns
+        -------
+        Series of objects
+        """
+        return _wrap_accessor_pandas(
+            self,
+            lambda x: x.str.center(width, fillchar),
+            StringType()
+        ).alias(self.name)
+
+    def contains(self, pat, case=True, flags=0, na=np.NaN, regex=True) -> 'ks.Series':
+        """
+        Test if pattern or regex is contained within a string of a Series.
+
+        Return boolean Series based on whether a given pattern or regex is
+        contained within a string of a Series.
+
+        Analogous to :func:`match`, but less strict, relying on
+        :func:`re.search` instead of :func:`re.match`.
+
+        Parameters
+        ----------
+        pat : str
+            Character sequence or regular expression.
+        case : bool, default True
+            If True, case sensitive.
+        flags : int, default 0 (no flags)
+            Flags to pass through to the re module, e.g. re.IGNORECASE.
+        na : default NaN
+            Fill value for missing values.
+        regex : bool, default True
+            If True, assumes the pat is a regular expression.
+            If False, treats the pat as a literal string.
+
+
+        Returns
+        -------
+        Series of boolean values
+            A Series of boolean values indicating whether the given pattern is
+            contained within the string of each element of the Series.
```

Comment:
while we're here, can we copy and paste Pandas doctests too? e.g., https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.str.contains.html?highlight=contains#pandas.Series.str.contains

The doctests don't need to be exactly same if it doesn't work currently.

Reply:
I don't mind if you want to do it separately in another PR.

Reply:
Added doctests for all the methods doctest-ed in Pandas and some more.